wit/wavs_worker.wit

```diff
@@ -8,6 +8,10 @@ use wasi:http/types@0.2.0 as http-types;
 use wasi:http/outgoing-handler@0.2.0 as http-outgoing-handler;
 
 interface layer-types {
+  record timestamp {
+    seconds: u64
```

Comment:
I think this should be nanoseconds, since we have that fidelity on cosmos chains. wdyt?